poetry.toml

```diff
@@ -0,0 +1,3 @@
+[virtualenvs]
+in-project = true
+create = true
```

Comment:
**nitpick** This is the default (https://python-poetry.org/docs/configuration/#virtualenvscreate) so not really needed to specify